src/AccessToken.php

```diff
@@ -208,7 +212,8 @@ private function verifyEs256($token, array $certs, $audience = null)
         }
 
         // @see https://cloud.google.com/iap/docs/signed-headers-howto#verifying_the_jwt_payload
-        if (!isset($payload['iss']) || $payload['iss'] !== self::IAP_ISSUER) {
+        $issuer = isset($issuer) ? $issuer : self::IAP_ISSUER;
```

Comment:
nit: this can be contracted:

```suggestion
        $issuer = $issuer ?: self::IAP_ISSUER;
```